src/groups/mqb/mqbu/mqbu_messageguidutil.t.cpp

```diff
@@ -318,9 +314,7 @@ static void test4_defaultHashUniqueness()
 
     mwctst::TestHelper::printTestName("DEFAULT HASH UNIQUENESS");
 
-#ifdef BSLS_PLATFORM_OS_AIX
```

Comment:
@alexander-e1off for awareness, might need to update your PR
https://github.com/bloomberg/blazingmq/pull/373/files